docs/tutorials/projects/filtered-camera-module.md

```diff
@@ -0,0 +1,407 @@
+1. The camera component displays the raw camera feed, but the transform camera will additionally overlay a bounding box on the same feed if a matching object is detected.
+   Try placing an object your ML model can recognize in front of the camera.
+   The transform camera should draw a bounding box around that object in the live camera feed, and indicate a confidence threshold for the match.
+
+   {{< imgproc src="/tutorials/filtered-camera-module/transform-camera-overlay.png" alt="The control tab in the Viam app showing both a live camera feed and the live transform camera overlay, with the latter correctly detecting a viam figure with a confidence score of 0.97" resize="800x" >}}
+
+1. When satisfied that your ML model is working well, you can disable both cameras.
+   Alternatively, if the transform camera is not matching reliably, consider adding and labelling more images in your dataset, or lowering the `confidence_threshold` of the transform camera.
+   Ideally, you want your ML model to be able to identify objects with a high level of confidence, which usually is dependent on a robust source dataset.
+
+## Add and configure the `filtered-camera` module
+
+With all the prerequisites in place, you are ready to add the `filtered-camera` module to your machine.
+
+1. On your machine's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Components** tab.
+1. Click the **Create component** button at the bottom of the page, select **Camera**, then select the `filtered-camera` model.
+   You can also search for `filtered-camera` directly.
+1. Give the modular camera a name, like `my-filtered-camera`, then click **Create**.
+1. On the panel that appears, enter the following configuration into the **Attributes** field:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "vision": "my-vision-service",
+     "window_seconds": 10,
+     "objects": {
+       "viam-figure": 0.6
+     },
+     "camera": "my-webcam"
+   }
+   ```
+
+   This example configures the `filter-camera` module to require a 60% confidence (`0.6`) threshold match for the label `viam-figure`, meaning that it must be at least 60% confident based on your trained ML model that the image contains the labelled object in order to capture it.
+   Images that do not meet this threshold do not trigger a successful match.
+
+   Additionally, the example configures a `window_seconds` value of `10` seconds, which controls the duration of a buffer of images captured _previous_ to a successful match.
+   With this configuration, images captured up to `10` seconds before the successful match are included in the capture and sync process.
+
+   If you used a different label when drawing your bounding boxes and training your ML model, supply it here instead of `viam-figure`.
```

Comment:
```suggestion
   If your model uses different labels - if you trained your model, the labels are the ones you assigned when drawing your bounding boxes - supply it here instead of `viam-figure`.
```

Reply:
Some tweaks here too: linking back to relevant sections.